packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/classes_table_diff.dart

```diff
@@ -171,7 +171,6 @@ class _InstanceColumn extends ColumnData<DiffClassStats>
       theHeap,
       data.heapClass,
       isSelected: isRowSelected,
-      gaContext: gac.MemoryAreas.snapshotDiff,
     );
```

Comment:
here and below: @polina-c this looks like a bug that we are not passing this through to the context menu button

Reply:
The context is supposed to be consumed by InstanceSetButton, that is under construction. @incendial , is there a way to exempt  gaContext from clean up?

Reply:
There are several ways to tell the command to ignore some code:
- via `--exclude` config to ignore a whole file
- via `ignore_for_file: unused_code`
- via `ignore: unused_code`

but I think I can just revert this removal manually and this should be enough.

Reply:
But my idea is to make the tests green, then you'll probably point out some other places that should be reverted and I want to apply all the review comments at once. Does it sound good?

Reply:
Sure. Yes, you can revert the change. Or, you can add `ignore: unused_code` for  gaContext in InstanceSetButton.

Reply:
I've reverted all changes with gaContext